src/Admin.php

```diff
@@ -31,9 +31,9 @@ public static function init() {
     add_action('woocommerce_variation_options_pricing', __CLASS__ . '::woocommerce_variation_options_pricing', 10, 3);
 
     // Generates price label PDF document.
-    add_action('post_action_label', __NAMESPACE__ . '\Label::post_action_label');
+    // add_action('post_action_label', __NAMESPACE__ . '\Label::post_action_label');
```

Comment:
Should this line be removed or uncommented?

Reply:
Fixed.